src/main/java/seedu/momentum/logic/parser/SortCommandParser.java

```diff
@@ -0,0 +1,78 @@
+package seedu.momentum.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.momentum.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.momentum.logic.commands.SortCommand.ASCENDING_ORDER;
+import static seedu.momentum.logic.commands.SortCommand.DESCENDING_ORDER;
+import static seedu.momentum.logic.parser.CliSyntax.SORT_ORDER;
+import static seedu.momentum.logic.parser.CliSyntax.SORT_TYPE;
+
+import seedu.momentum.logic.commands.SortCommand;
+import seedu.momentum.logic.parser.exceptions.ParseException;
+import seedu.momentum.model.project.SortType;
+
+public class SortCommandParser implements Parser<SortCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the SortCommand
+     * and returns a SortCommand object for execution.
+     * @throws ParseException if the user does not conform to the expected format.
+     */
+    public SortCommand parse(String args) throws ParseException {
```

Comment:
Just checking, what's the expected behaviour when there is a preamble?

For example, `sort hello type/alpha order/dsc`

Reply:
It should disregard the preamble. 
I have added a test for this!

Reply:
I have changed the SortCommandParser to fail when there is a non-empty preamble for consistency as AddCommandParser and FindCommandParser does the same. I have added a test accordingly!